Formula/m/meson.rb

```diff
@@ -13,10 +13,10 @@ class Meson < Formula
 
   depends_on "python-setuptools" => :build
   depends_on "ninja"
-  depends_on "python@3.12"
```

Comment:
if this moves back to py3,.11, then it does not need `depends_on "python-setuptools" => :build`